src/Pos/Aeson/Types.hs

```diff
@@ -42,6 +43,9 @@ instance ToJSON TxOut where
         "coin"    .= coinToInteger txOutValue,
         "address" .= sformat build txOutAddress ]
 
+deriving instance ToJSON SlotAmount
+deriving instance ToJSON BlockAmount
+
 -- NOTE: some of these types are used on frontend (PureScript).
```

Comment:
Probably not because `deriving` will use GND here and so the instance will be the same as for the underlying type.